.circleci/local_publish_helpers.sh

```diff
@@ -59,7 +59,10 @@ function generatePkgCli {
   yarn --production
 
   # Optimize package size
-  yarn rimraf **/*.d.ts **/*.js.map **/*.d.ts.map **/README.md **/readme.md **/Readme.md **/CHANGELOG.md **/changelog.md **/Changelog.md **/HISTORY.md **/history.md **/History.md
+  find . \
+    -name "*.d.ts" -or -name "*.js.map" -or -name "*.d.ts.map" -or \
+    -iname "readme.md" -or -iname "changelog.md" -or -iname "history.md" \
+    | xargs rm
```

Comment:
Changed because the globs were giving "too many arguments" type errors on multiple platforms